asv_bench/benchmarks/timeseries.py

```diff
@@ -1059,33 +1059,26 @@ class timeseries_to_datetime_iso8601(object):
     goal_time = 0.2
 
     def setup(self):
-        self.N = 100000
-        self.rng = date_range(start='1/1/2000', periods=self.N, freq='T')
-        if hasattr(Series, 'convert'):
-            Series.resample = Series.convert
-        self.ts = Series(np.random.randn(self.N), index=self.rng)
         self.rng = date_range(start='1/1/2000', periods=20000, freq='H')
-        self.strings = [x.strftime('%Y-%m-%d %H:%M:%S') for x in self.rng]
+        self.strings = self.rng.strftime('%Y-%m-%d %H:%M:%S')
+        self.strings_nosep = self.rng.strftime('%Y%m%d %H:%M:%S')
+        self.strings_tz_space = pd.Series(self.rng.strftime('%Y-%m-%d %H:%M:%S')) + ' -0800'
```

Comment:
The `strftime` method on a DatetimeIndex is rather new. I don't really know what our policy on this should be, but I think this will make it difficult to run the asv benchmarks for a longer time back?